go/vt/schemadiff/schema.go

```diff
@@ -241,10 +253,12 @@ func (s *Schema) normalize() error {
 				}
 				referencedEntity, ok := s.named[referencedTableName]
 				if !ok {
-					return &ForeignKeyNonexistentReferencedTableError{Table: name, ReferencedTable: referencedTableName}
+					errs = errors.Join(errs, addEntityFkError(t, &ForeignKeyNonexistentReferencedTableError{Table: name, ReferencedTable: referencedTableName}))
```

Comment:
I was going to ask if we wanted to add/join nil errors as well, but then saw the docs: https://pkg.go.dev/errors#Join :-)